slices/coreutils.yaml

```diff
@@ -0,0 +1,120 @@
+package: coreutils
+
+slices:
+  bins:
+    essential:
+      - libacl1_libs
+      - libattr1_libs
+      - libc6_libs
+      - libcrypt1_libs
+      - libgcc-s1_libs
+      - libgmp10_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+    contents:
+      /bin/cat:
+      /bin/chgrp:
+      /bin/chmod:
+      /bin/chown:
+      /bin/cp:
+      /bin/date:
+      /bin/dd:
+      /bin/df:
+      /bin/dir:
+      /bin/echo:
+      /bin/false:
+      /bin/ln:
+      /bin/ls:
+      /bin/mkdir:
+      /bin/mknod:
+      /bin/mktemp:
+      /bin/mv:
+      /bin/pwd:
+      /bin/readlink:
+      /bin/rm:
+      /bin/rmdir:
+      /bin/sleep:
+      /bin/stty:
+      /bin/sync:
+      /bin/touch:
+      /bin/true:
+      /bin/uname:
+      /bin/vdir:
+      /usr/bin/[:
+      /usr/bin/arch:
+      /usr/bin/b2sum:
+      /usr/bin/base32:
+      /usr/bin/base64:
+      /usr/bin/basename:
+      /usr/bin/basenc:
+      /usr/bin/chcon:
+      /usr/bin/cksum:
+      /usr/bin/comm:
+      /usr/bin/csplit:
+      /usr/bin/cut:
+      /usr/bin/dircolors:
+      /usr/bin/dirname:
+      /usr/bin/du:
+      /usr/bin/env:
+      /usr/bin/expand:
+      /usr/bin/expr:
+      /usr/bin/factor:
+      /usr/bin/fmt:
+      /usr/bin/fold:
+      /usr/bin/groups:
+      /usr/bin/head:
+      /usr/bin/hostid:
+      /usr/bin/id:
+      /usr/bin/install:
+      /usr/bin/join:
+      /usr/bin/link:
+      /usr/bin/logname:
+      /usr/bin/md5sum:
+      /usr/bin/mkfifo:
+      /usr/bin/nice:
+      /usr/bin/nl:
+      /usr/bin/nohup:
+      /usr/bin/nproc:
+      /usr/bin/numfmt:
+      /usr/bin/od:
+      /usr/bin/paste:
+      /usr/bin/pathchk:
+      /usr/bin/pinky:
+      /usr/bin/pr:
+      /usr/bin/printenv:
+      /usr/bin/printf:
+      /usr/bin/ptx:
+      /usr/bin/realpath:
+      /usr/bin/runcon:
+      /usr/bin/seq:
+      /usr/bin/sha1sum:
+      /usr/bin/sha224sum:
+      /usr/bin/sha256sum:
+      /usr/bin/sha384sum:
+      /usr/bin/sha512sum:
+      /usr/bin/shred:
+      /usr/bin/shuf:
+      /usr/bin/sort:
+      /usr/bin/split:
+      /usr/bin/stat:
+      /usr/bin/stdbuf:
+      /usr/bin/sum:
+      /usr/bin/tac:
+      /usr/bin/tail:
+      /usr/bin/tee:
+      /usr/bin/test:
+      /usr/bin/timeout:
+      /usr/bin/tr:
+      /usr/bin/truncate:
+      /usr/bin/tsort:
+      /usr/bin/tty:
+      /usr/bin/unexpand:
+      /usr/bin/uniq:
+      /usr/bin/unlink:
+      /usr/bin/users:
+      /usr/bin/wc:
+      /usr/bin/who:
+      /usr/bin/whoami:
+      /usr/bin/yes:
+      /usr/libexec/coreutils/libstdbuf.so:
```

Comment:
I think this should be in a `libs` slice.You can then add that slices as an `essential` in the `bins` slice.

Reply:
I'm uncertain if we should place this library in a separate `libs` section. It appears to be an internal library for `coreutils` and isn't meant for external use.

Reply:
> It appears to be an internal library for `coreutils` and isn't meant for external use.

True. But typically we group the libraries (including internal ones) into `libs` and binaries into `bins` for better categorization.

Reply:
Make sense, updated, thanks!